apps/block_scout_web/assets/js/lib/smart_contract/consensus.js

```diff
@@ -0,0 +1,52 @@
+const CONSENSUS_ADDRESS = '0x3014ca10b91cb3D0AD85fEf7A3Cb95BCAc9c0f79'
```

Comment:
`PROVIDER_URL` and `CONSENSUS_ADDRESS` should not be hard coded but taken from ENV param
See this doc: https://github.com/fuseio/blockscout/tree/master/docker
and the makefile code: https://github.com/fuseio/blockscout/blob/master/docker/Makefile